nixos/modules/services/audio/ympd.nix

```diff
@@ -15,7 +15,7 @@ in {
       enable = mkEnableOption "ympd, the MPD Web GUI";
 
       webPort = mkOption {
-        type = types.string;
+        type = types.either types.string types.port; # string for backwards compat
```

Comment:
Oh actually one more thing, while you're changing this part of the code, you can also switch to `types.str`, as `types.string` is deprecated.

Reply:
:+1: